openid-4-verifiable-presentations-1_0.md

```diff
@@ -261,6 +268,13 @@ This specification defines the following new parameters:
 
 A public key to be used by the Wallet as an input to the key agreement to encrypt Authorization Response (see (#jarm)). It MAY be passed by the Verifier using the `jwks` or the `jwks_uri` claim within the `client_metadata` or `client_metadata_uri` request parameter.
 
+`request_uri_method`: 
+: OPTIONAL. A string determining the HTTP method to be used with the `request_uri` included in the same request. Two case-sensitive values are defined for `request_uri_method` in this specification: `get` and `post`. If `request_uri_method` is `get`, the Wallet MUST send the request to retrieve the Request Object using the HTTP GET method, i.e., as defined in [@RFC9101]. If `request_uri_method` is `post`, a supporting Wallet MUST send the same request using the HTTP POST method as detailed in (#request_uri_method_post). 
+`request_uri_method` MUST only be present if the request contains a `request_uri` parameter. If the `request_uri_method` parameter is not present, the Wallet MUST process the `request_uri` as defined in [@RFC9101]. Wallets not supporting the method `post` will send a GET request to the request URI (default behavior as defined in [@RFC9101]).  
```

Comment:
```suggestion
`request_uri_method`: 
: OPTIONAL. A string determining the HTTP method to be used when the `request_uri` parameter is included in the same request. Two case-sensitive valid values are defined in this specification: `get` and `post`. If `request_uri_method` value is `get`, the Wallet MUST send the request to retrieve the Request Object using the HTTP GET method, i.e., as defined in [@RFC9101]. If `request_uri_method` value is `post`, a supporting Wallet MUST send the request using the HTTP POST method as detailed in (#request_uri_method_post). If the `request_uri_method` parameter is not present, the Wallet MUST process the `request_uri` parameter as defined in [@RFC9101]. Wallets not supporting the `post` method will send a GET request to the request URI (default behavior as defined in [@RFC9101]). `request_uri_method` parameter MUST NOT be present if a `request_uri` parameter is not present.
```

I don't understand what `the same request` means in "a supporting Wallet MUST send the same request using the HTTP POST method". same as JAR (but it's not the same AFAIK)?

explanation of the same parameter should probably not be across multiple paragraphs. need to double check, but i don't think it gets rendered as belonging to the same bulletpoint.

I would make it clear that `request_uri_method` parameter without a `request_uri` parameter is an error condition. I would also move that sentence to the end because it is inbetween the sentences that describe what to to do when the value is get/post and what if the value is present/wallet does not support it (which should be logically next to each other iMO).

I once had to do an editorial PR adding `parameter` after the parameter names  which were just in the text, so would appreciate if we can be consistent on "`request_uri_method` parameter" or "request URI method" (which would need to be defined.)